internal/provisioner/kops_provisioner.go

```diff
@@ -114,8 +116,17 @@ func (provisioner *KopsProvisioner) CreateCluster(cluster *model.Cluster, aws aw
 	}
 
 	err = os.Rename(kops.GetOutputDirectory(), outputDir)
-	if err != nil {
-		return errors.Wrap(err, fmt.Sprintf("failed to rename kops output directory to %q", outputDir))
+	if err != nil && err.(*os.LinkError).Err == syscall.EXDEV {
+		err = utils.CopyDirectory(kops.GetOutputDirectory(), outputDir)
+		if err != nil {
+			return fmt.Errorf("failed to rename kops output directory to %q using utils.CopyFolder", outputDir)
+		}
+		err = os.RemoveAll(kops.GetOutputDirectory())
```

Comment:
We actually already cleanup this dir with the `defer kops.Close()` line above.

This is what `kops.Close()` does:

```
// Close cleans up the temporary output directory used by kops.
func (c *Cmd) Close() error {
	return os.RemoveAll(c.tempDir)
}
```

Reply:
@gabrieljackson ah, so we dont need that?

Reply:
Yes, exactly.

Reply:
will remove, thanks!

Reply:
@gabrieljackson done